cpp.md

```diff
@@ -311,75 +319,7 @@ int main()
 }
 
 ```
-## Stacks 
```

Comment:
Why this got deleted?

Reply:
As stack and queue are one of the DS only, so only including these two seemed incomplete and inconsistent. If you say, I'll undo that change.